public/app.js

```diff
@@ -0,0 +1,330 @@
+// AUTH CONTROLLER
+var authController = (function() {
+    return {
+        //Authentication state listener
+        initApp: function() {
+            firebase.auth().onAuthStateChanged(function (user) {
+                if (user) {
+                    console.log('Login Successful');
+                    console.log("Attempting to bind: " + user.email)
+                    console.log('Bind Successful');
+                    return user;
+                } else {
+                    window.location.replace('auth.html')
+                }
+            });
+        },
+
+        //logout function
```

Comment:
I don't think this comment is helpful. It's pretty obvious what `logout: function()` does and what its type is

Reply:
I'll remove the comment